### PR TITLE
net-irc/quasselgrep: bump to 0_p20200830, new proxy maintainer

### PR DESCRIPTION
**Pull request by jsmolic**
```diff
diff quasselgrep-0_p20190211.ebuild quasselgrep-0_p20200830.ebuild
6c6
< PYTHON_COMPAT=( python3_{6,7} )
---
> PYTHON_COMPAT=( python3_{6,7,8} )
7a8
> DISTUTILS_USE_SETUPTOOLS=rdepend
14,23c15,18
< if [[ "${PV}" == "9999" ]] ; then
< 	EGIT_REPO_URI="https://github.com/65278/quasselgrep.git"
< 	inherit git-r3
< 	KEYWORDS=""
< else
< 	MY_COMMIT=9b6b0bc1252daa6e574363d87d04eebd981215a5
< 	SRC_URI="https://github.com/fish-face/${PN}/tarball/${MY_COMMIT} -> ${P}.tar.gz"
< 	S="${WORKDIR}/fish-face-${PN}-${MY_COMMIT:0:7}"
< 	KEYWORDS="~amd64 ~x86"
< fi
---
> MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
> SRC_URI="https://github.com/fish-face/quasselgrep/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
> S="${WORKDIR}/${PN}-${MY_COMMIT}"
> KEYWORDS="~amd64 ~x86"
29a25
> 	dev-python/psycopg:2[${PYTHON_USEDEP}]
31,34d26
<
< pkg_postinst() {
< 	optfeature "access postgres db" dev-python/psycopg:2
< }
```

### CODE REVIEW

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```
Comment (DarthGandalf) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:6]: Does it support 3.8?

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+PYTHON_REQ_USE=( sqlite )
+
+inherit distutils-r1 eutils python-r1
```
Comment (DarthGandalf) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:9]: No need to inherit python when inheriting distutils

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+DESCRIPTION="Tool for searching quassel logs from the commandline"
+HOMEPAGE="https://github.com/fish-face/quasselgrep"
+
+MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
+SRC_URI="https://github.com/fish-face/${PN}/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (DarthGandalf) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:15]: No PN in SRC_URI please
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+S="${WORKDIR}/fish-face-${PN}-${MY_COMMIT:0:7}"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="sqlite postgres"
```
Comment (DarthGandalf) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:21]: https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild (hunk 1 above)
Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Yes, I added that

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild (hunk 2 above)
Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Done, thanks

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild (hunk 3 above)
Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Eh, I knew that, somehow I missed it while I was working on this... I updated it. It's always good to have a pair of fresh eyes take a look et the ebuild ;)
Thanks!

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild (hunk 4 above)
Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Done

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 5]
```diff
@@ -3,32 +3,24 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (thesamesam) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:6]: 3.9? If deps in the way, file bugs if they don't exist -- or even fix them too ;)

Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Deps are okay, py3.9 is added

Comment (thesamesam) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: If tests pass and it works, cool.

Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: Works with 3.9, there are no tests AFAICS

File: net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild [hunk 6]
```diff
@@ -3,32 +3,24 @@
...
+SRC_URI="https://github.com/fish-face/quasselgrep/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${MY_COMMIT}"
+KEYWORDS="~amd64 ~x86"
 
 LICENSE="GPL-2"
 SLOT="0"
 
 RDEPEND="dev-python/future[${PYTHON_USEDEP}]
 	dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/psycopg:2[${PYTHON_USEDEP}]
```
Comment (thesamesam) [inline net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild:25]: I don't love dragging in PostgreSQL unconditionally. Is there a reason you dropped the optfeature?

IIRC you can use Quassel with other backends.

Comment (jsmolic) [reply net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild]: No, I should have left the optfeature, apologies. I have updated it now.

**Pull request closed without merging**